docs/en/sql-reference/statements/explain.md

```diff
@@ -412,4 +417,37 @@ Result:
 └──────────┴───────┴───────┴──────┴───────┘
 ```
 
+### EXPLAIN TABLE OVERRIDE {#explain-table-override}
+
+Shows the result of a table override on a table schema accessed through a table function.
+Also does some validation, throwing an exception if the override would have caused some kind of failure.
+
+**Example**
+
+Assume you have a remote MySQL table like this:
+
+```sql
+CREATE TABLE db.tbl (
+    id INT PRIMARY KEY,
+    created DATETIME DEFAULT now()
+)
+```
+
+```sql
+EXPLAIN TABLE OVERRIDE mysql('127.0.0.1:3306', 'db', 'tbl', 'root', 'clickhouse')
```

Comment:
Can you please add integration test for this?
So if the code gets broken accidentally by someone, it will no go unnoticed.

Reply:
Will do!